File: nodejs/7 kyu/145.js
// https://www.codewars.com/kata/59cd0535328801336e000649/train/javascript

// Passed

function interest(p, r, n) {
  let totalUnderSimpleInterest = p * (1 + r * n);
  let totalUnderCompoundInterest = p * (1 + r) ** n;

  let result = [
    Math.round(totalUnderSimpleInterest),
    Math.round(totalUnderCompoundInterest),
  ];
  return result;
}
const output = interest(100, 0.1, 2);
console.log(output);

File: python/6 kyu/145.py
# https://www.codewars.com/kata/53689951c8a5ca91ac000566/train/python

# Passed

def args_to_string(args):
    result = str()
    for arg in args:
        if type(arg) == str:
            result += arg
        elif type(arg) == list:
            if len(arg) > 1:
                if len(arg[0]) > 1:
                    prefix = "--"
                else:
                    prefix = "-"

                result += prefix + " ".join(arg)
            else:
                result += arg[0]

        result += " "
    
    result = result.strip()
    return result
            
output = args_to_string([["foo", "bar"], "baz"])
print(output)